lib/services/unified_resource.go

```diff
@@ -475,6 +431,12 @@ func (c *UnifiedResourceCache) itemKindMatches(r resource, kinds map[string]stru
 		_, ok := kinds[types.KindSAMLIdPServiceProvider]
 		return ok
 	case types.KindAppServer:
+		if r.(types.AppServer).GetApp().GetSubKind() == types.KindIdentityCenterAccount {
```

Comment:
Is this the only subkind we have to be worried about?

Reply:
Just fyi that the `saml_idp_service_provider` kind is also converted to "apps" but that is done in the proxy

https://github.com/gravitational/teleport/blob/280e1f5f6f9e16ae954153663455a3ae9101e877/lib/web/apiserver.go#L3198-L3207

Reply:
Yeah it does not impact current changes in any way.
Also doing conversion in auth is a better option and is something we will need to do to support resource listing in `tsh app ls` which saml and this identity center account resource does not support currently.